### ISSUE TITLE
{-# LANGUAGE CPP #-} does not strip c-style comments

### ISSUE THREAD

**Issue by hvr** [ndmitchell/hlint#21]
Haskell code that `#include`s header files, usually has to cope with c-style comments.
E.g. the following simple program:

``` haskell
{-# LANGUAGE CPP #-}

#include "MachDeps.h"

isBigEndian :: Bool
#if WORDS_BIGENDIAN
isBigEndian = True
#else
isBigEndian = False
#endif

wordSize :: Int
#if (WORD_SIZE_IN_BITS == 64) || (WORD_SIZE_IN_BITS == 32)
wordSize = WORD_SIZE_IN_BITS
#else
#error unsupported word size
#endif

main = do
  putStrLn $ "big-endian = " ++ show isBigEndian
  putStrLn $ "word-size = " ++ show wordSize ++ " bits"
```

However, I can't seem to find a way/CLI-flag to tell `hlint` to instruct `cpphs` to strip c-comments. So `hlint` complains about parsing errors as soon as a c-style comment is encountered.
